components/organisms/BlogSection/blog-section.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from 'react';
+
+import { DevToArticleResults } from './blog-section.types';
+import { BlogCard } from '@components/atoms';
+
+export function BlogSection() {
+    const [blogData, setBlogData] = useState<DevToArticleResults[] | []>([]);
+
+    useEffect(() => {
```

Comment:
Might be good to out a check in before this fetch that you don't already have the data. That why you won't do unnecessary fetches if the component re-renders